src/Components/Charts/Samples/BarChart/BarChart.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+    BarChart, Bar, XAxis, YAxis, CartesianGrid, Tooltip, Legend, ResponsiveContainer,
+} from 'recharts';
+
+import Typography from '@material-ui/core/Typography';
+
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import styles from './_styles';
+
+const useStyles = makeStyles(styles);
+
+const parseDataset = (datasetComponent) => {
```

Comment:
I guess it is similar to: 
"const parseDataset = ({props}) => {"

Reply:
changed